engine/render/src/render/render.cpp

```diff
@@ -38,6 +38,8 @@ namespace dmRender
     using namespace dmVMath;
 
     const char* RENDER_SOCKET_NAME = "@render";
+    const char* KEY_CONTEXT_LOST = "context_lost";
+    const char* KEY_CONTEXT_RESTORED = "context_restored";
```

Comment:
Do we need to have these as strings instead of an int/enum?

Reply:
I'd prefer int/enum if possible. (Smaller exe size as well as less prone to typos)

Reply:
Replace with `dmRender::RenderContextEvent` enumeration. On JS side add 2 field with values for that enumeration. I didn't use `embind` to use constants from C++ because it requires add `-lembind` to linker (and as I understand - add more size to result build).